application/constants/http.py

```diff
@@ -0,0 +1,37 @@
+"""
+Helm constants.
+"""
+from .base_enum import StrEnum
+
+
+class HttpStatuses(StrEnum):
+    """
+    Helm release statuses.
```

Comment:
Helm? Did you revised statuses? How HTTP component can have status `pending-install` or `uninstalled`?